tests/arch/arm/arm_runtime_nmi/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
-  arch.interrupt:
+  arch.interrupt.arm.nmi:
```

Comment:
can you name it arch.arm.nmi?

Reply:
Yes, I will rename it. Could you provide algorithm, how Zephyr community defines names of the cases? It seems that I have 30 more cases waiting to be renamed, so it will be great to know standard in naming of the cases. Thanks.

Reply:
No I don't think so. You can probably be safe with using the name that matches the directory path.

Reply:
> No I don't think so. You can probably be safe with using the name that matches the directory path.

no please, we need to keep some hierarchy here, path of the test is completely unrelated, tests move around...

so arch.interrupt was correct here. The end goal is to create the testsuite and update it using the output from sanitycheck.

Reply:
> can you name it arch.arm.nmi?

keep arch.interrupt to maintain hierarchy and possibility to file all of those tests under Interrupts in the testsuite.